core/templates/app.constants.ajs.ts

```diff
@@ -18,12 +18,12 @@
 
 // TODO(#7092): Delete this file once migration is complete and these AngularJS
 // equivalents of the Angular constants are no longer needed.
-import { AppConstants } from 'app.constants';
-
 import constants from 'assets/constants';
+import { AppConstants } from 'app.constants';
```

Comment:
Why are there changes in this file, they do not seem to do anything?

Reply:
done, delete this change.